tensorflow-serving/meta.yaml

```diff
@@ -8,8 +8,8 @@ source:
    git_url: https://github.com/tensorflow/serving.git
    git_rev: {{ version }}
    patches:
-     - 0000a-tensorflow_patches.patch        # [x86_64]
-     - 0000b-Add-memcpy-wrapper-patch.patch  # [x86_64 and build_type == 'cuda' and ((cudatoolkit=='10.2' and py<38) or (cudatoolkit=='11.0'))]
+     - 0000a-TF-Build-fix-x86-64.patch       # [x86_64 and (build_type == 'cpu' or (build_type == 'cuda' and cudatoolkit == '10.2' and py == 38))]
+     - 0000b-Fix-plus-memcpy-patch.patch     # [x86_64 and build_type == 'cuda' and ((cudatoolkit == '10.2' and py<38) or (cudatoolkit == '11.0'))]
      - 0001-TF-Build-fix-ppc64le.patch       # [ppc64le]
  
 build:
```

Comment:
We would also want to update the selector while specifying requirements in meta.yaml.

Reply:
No changes should be needed for the reqs